tests/hypertext/HTMLLintBearTest.py

```diff
@@ -9,21 +9,47 @@
 </html>
 """
 
+invalid_test = """
+<html>
+       <body>
+    <p class='foo'>
+    <h1>Hello, world!</h1>
+    <br> </br>
+  </body>
+</html>
+"""
+
 HTMLLintBearTest = verify_local_bear(HTMLLintBear,
-                                     valid_files=(),
+                                     valid_files=('',),
```

Comment:
Is this necessary for the feature? Otherwise different commit, testing an empty string is definitely not bad :+1:

Reply:
bump :)

Reply:
I'm sorry I didn't get the meaning of the last comment.. should I remove the changes mentioned above regarding the empty string, make another issue and implement the same there?

Reply:
just curious why you now introduced this empty string. If this can be separated into a new commit, this would ease review.

Reply:
In https://github.com/coala/coala-bears/pull/1304/files one of the reviewers mentioned it's bad practice to leave the parameter for valid/invalid file to be empty.Hence I changed that here, no other particular reason

Reply:
Then, move this into a new commit please :D You can just put it ontop in this PR :+1:

Reply:
Exactly write this reason into the new commit message ;D